Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -998,6 +998,12 @@ class DropDownMenu
     }
   }
 
+  private static SetEventKeyCode(event: KeyboardEvent, keyCode: number) {
+    Object.defineProperties(event, {
+      keyCode: {value: keyCode}
+    });
+  }
```

Comment:
I dislike that we are overriding an existing property of a browser object. While it seems to work fine in the relevant browsers there is no guarantee that it stays that way. Therefore, I would refrain from changing `Event`.

Instead I suggest creating a new object instead of changing the existing one. Since we only call `Options_OnKeyDown` with the changed event we can change the parameter type so that we can use structural typing for the properties that we need. We only need the three properties `keyCode`, `preventDefault`, and `stopPropagation`, which we can define in a interface/type. By using this new type as parameter type we can pass in an `Event` or custom object with those three properties.

Kind of difficult to explain, maybe we get together to change the code :)